frontend/src/features/utils/views/ComponentsView.tsx

```diff
@@ -13,11 +14,22 @@ const ComponentsView = () => (
     <Link to="/">Wróć do strony głównej</Link>
     <hr/>
     <Item>
+      Button component
+      <br></br>
       <Button>Główny przycisk</Button>
     </Item>
+
     <Item>
+      <br></br>
+      Header component
       <Header></Header>
     </Item>
+    
+    <Item>
+      <br></br>
+      Card component
```

Comment:
Opis może zostać, ale też sprawdź sam komponent Card i przekaż mu jakieś propsy.

Zrób to w taki sposób:

`
<img src="#" onerror="alert('GitHub coś ssie teraz')" />
<Card title="Nagłówek" extra="Jakaś dodatkowa zawartość">
<div>Tutaj jakaś treść</div>
</Card>
`

Zobacz sobie jak wygląda i jakie możliwości ma ten komponent: [https://ant.design/components/card/](https://ant.design/components/card/)

Reply:
Tymi backslashami się nie przejmuj, GitHub coś ssie i nie można normalnie znaczników wprowadzać